docs/reference-guide/triggers.md

```diff
@@ -48,7 +43,7 @@ SET object.updated_at = timestamp();
 The query may seem complex, so let's break it down:
 * `CREATE TRIGGER exampleTrigger`: This statement creates the trigger.
 * `ON UPDATE AFTER COMMIT EXECUTE`: This statement specifies what kind of event
-  should activate the trigger. This one will be triggered for every update
+  should activate the execution of trigger. This one will be triggered for every update
   operation and the query below will be executed after the update event has been
   committed.
 * `UNWIND updatedObjects AS updatedObject`: If multiple objects were updated,
```

Comment:
I would add a part of the query where you define the name just so we are 100% clear what we are talking about